### ISSUE TITLE
Address challenges with location-restricted configuration and implementation

### ISSUE THREAD

**Issue by dinahhandel** [sul-dlss/media#158]
The way that location-based access is currently configured is tenuous because it relies upon IP addresses for the workstations in specific reading rooms, as well as Ops-configured ethernet set-ups for specific SUL staff located outside of the reading rooms. When the workstation operating system is updated or settings change, it is possible that reading room staff will have to reconfigure access. Furthermore, there are many SUL staff who would benefit from having access to location-restricted media outside of the location, but are unable to have access configured. 

The desired outcome is to address both issues: the configuration of location-restricted access should not be burdensome for reading room staff to set-up and maintain and some SUL staff (digitization lab, special collections) should be able to view location-restricted media from their computers.

**Comment by dinahhandel** [comment]
Would be helpful to know from TLs @jcoyne @cbeer if this ticket is actionable. Per the kick-off inception deck it is considered unresolved (therefore not as high priority as "in" items), but it is also a ticket that does not require design AFAIK.

**Comment by dinahhandel** [comment]
@andrewjbtw and I agree that this needs a story time.

**Comment by cbeer** [comment]
Agreed, a story time and a discussion with ops and/or tech support is probably needed.

**Comment by mjgiarlo** [comment]
Mike will reach out to ops to figure out who we can talk to about this and then set up a meeting between ops, POs, TLs.

**Comment by mjgiarlo** [comment]
We'll discuss this on Nov. 10th, 2023.

**Comment by andrewjbtw** [comment]
One outcome of the Nov 10th meeting is that we'll be splitting this into two issues: one for stanford/public access and one for staff/admin access.

**Comment by dinahhandel** [comment]
For the staff/admin access, POs have decided to defer meeting with stakeholders until 2024. It is not expected that this ticket will be addressed in the current WC.
